demos/functions/views/demo/workbox-cache-expiration/sw.hbs

```diff
@@ -1,8 +1,5 @@
-importScripts('{{CDN_URL}}/workbox-sw.dev.js');
-
-workbox.setConfig({
-  debug: true
-});
+importScripts('{{CDN_URL}}/workbox-core.dev.js');
+importScripts('{{CDN_URL}}/workbox-cache-expiration.dev.js');
```

Comment:
Out of curiosity, why not use `workbox-sw` here?

Reply:
The cdn_url only applies to top level so Workbox-sw would need the different path passed in which might look a tad weird for published demos. Need to find a better to switch better CDN and local builds

Reply:
Opened an issue for changing how local and cdn builds are used.

https://github.com/GoogleChrome/workbox/issues/959